src/main/java/sa/assertj/iterables/AreExactly.java

```diff
@@ -0,0 +1,34 @@
+package sa.assertj.iterables;
+
+import org.assertj.core.api.Condition;
+import sa.assertj.Experiment;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AreExactly extends Experiment {
+    static Experiment.DataProvider provider = (size, numOfSamples) -> {
+        Object[][] result = new Object[numOfSamples][];
+        for (int i = 0; i < numOfSamples; i++) {
+            ArrayList<Integer> list = new ArrayList<>();
+            Random rand = new Random();
+            int element = rand.nextInt();
+            for (int j = 0; j < size; j++) {
+                list.add(element);
+            }
+
+            Condition<Integer> condition = new Condition<>(s -> s.equals(element), "equals");
+
+            result[i] = new Object[] { list, size, condition};
```

Comment:
Yeah, again, I think it's nicer if there is always a different amount of these, maybe we can make a condition like `< 0` and then count how many there are under zero before setting the `exactly`ness?